### DISCUSSION TITLE
Parallel script execution

### DISCUSSION THREAD

**Discussion by Maschinenausleger** [ansys/pyaedt#2365] [Q&A] [answered]
Hi I'm trying to simulate the same small maxwell 2D optimetrics (parametric with 4 configurations of the current) for multiple material configurations. Ansys allows parallelization of the optimetrics, but it would be much faster to execute every material combination in parallel. Is it somehow possible to run the script in parallel which adapted the material and starts the optimetrics? 

Thanks for you're answers
Chris

**Comment by Samuelopez-ansys** [comment, marked as answer]
@Maschinenausleger It is possible but you will need a new maxwell license for each different thread you launch in parallel (different material). Using 'thread' module I think should be possible.

**Reply by Samuelopez-ansys** [reply]
For now, it is not possible to run multiple AEDT processes in the same PyAEDT session, the problem in on AEDT.
